pkg/action/show_test.go

```diff
@@ -30,8 +30,8 @@ func TestShow(t *testing.T) {
 		Files: []*chart.File{
 			{Name: "README.md", Data: []byte("README\n")},
 			{Name: "crds/ignoreme.txt", Data: []byte("error")},
-			{Name: "crds/foo.yaml", Data: []byte("---\nfoo\n")},
```

Comment:
These tests are showing that a CRD yaml which already has `---` renders correctly

Reply:
True, fixed